From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 26 Jul 2018 18:28:45 +0000
Subject: [PATCH] Enable package to build

Get package to build with removing -Werror flag and adding a dummy
function.
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fe8f537..aa196dd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1222,7 +1222,6 @@ AC_C_VOLATILE
 
 ### General flags / actions
 CFLAGS="$CFLAGS -Wall"
-CFLAGS="$CFLAGS -Werror"
 
 # -Wno-unknown-pragmas is due to gcc not understanding '#pragma ident'
 # in Xlib.h on OpenSolaris.
